apps/opik-frontend/src/components/pages/TracesPage/MetricsTab/MetricsTab.tsx

```diff
@@ -0,0 +1,203 @@
+import { Button } from "@/components/ui/button";
+import { ChartLine as ChartLineIcon } from "lucide-react";
+import React, { useEffect, useMemo, useRef, useState } from "react";
+import SelectBox from "@/components/shared/SelectBox/SelectBox";
+import MetricChart from "@/components/pages/TracesPage/MetricsTab/MetricChart/MetricChart";
+import useProjectMetric, {
+  IntervalType,
+} from "@/api/projects/useProjectMetric";
+import last from "lodash/last";
+import dayjs from "dayjs";
+import { StringParam, useQueryParam, withDefault } from "use-query-params";
+import RequestChartDialog from "@/components/pages/TracesPage/MetricsTab/RequestChartDialog/RequestChartDialog";
+import useTracesOrSpansList, {
+  TRACE_DATA_TYPE,
+} from "@/hooks/useTracesOrSpansList";
+import NoTracesPage from "@/components/pages/TracesPage/NoTracesPage";
+
+const DAYS_OPTIONS = [
+  {
+    value: "1",
+    label: "1 day",
+  },
+  {
+    value: "3",
+    label: "3 days",
+  },
+  {
+    value: "7",
+    label: "7 days",
+  },
+  {
+    value: "30",
+    label: "30 days",
+  },
+];
+
+const POSSIBLE_DAYS_OPTIONS = DAYS_OPTIONS.map((dayOption) => dayOption.value);
```

Comment:
I think it would be cleaner to define constants (enum) for all these values and use them in `DAYS_OPTIONS` array.